Bug: http://bugzilla.clutter-project.org/show_bug.cgi?id=2256
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=580106

--- a/cogl/cogl-fixed.c
+++ b/cogl/cogl-fixed.c
@@ -626,7 +626,7 @@
 	/*
 	 * Find the highest bit set
 	 */
-#if __arm__
+#if 0
 	/* This actually requires at least arm v5, but gcc does not seem
 	 * to set the architecture defines correctly, and it is I think
 	 * very unlikely that anyone will want to use clutter on anything
