From 971c31bb3e13344a0203c0508119e81a4ea2c08b Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd@luon.net>
Date: Sat, 23 Feb 2013 17:29:25 +0100
Subject: [PATCH] Disable reliance on KHR_create_context

---
 cogl/winsys/cogl-winsys-egl.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/cogl/winsys/cogl-winsys-egl.c b/cogl/winsys/cogl-winsys-egl.c
index af8181b..dfce845 100644
--- a/cogl/winsys/cogl-winsys-egl.c
+++ b/cogl/winsys/cogl-winsys-egl.c
@@ -355,7 +355,9 @@ try_create_context (CoglDisplay *display,
           error_message = "Driver does not support GL 3 contexts";
           goto fail;
         }
-
+      error_message = "GL 3 driver not supported by this build";
+      goto fail;
+#if 0
       /* Try to get a core profile 3.1 context with no deprecated features */
       attribs[0] = EGL_CONTEXT_MAJOR_VERSION_KHR;
       attribs[1] = 3;
@@ -366,6 +368,7 @@ try_create_context (CoglDisplay *display,
       attribs[6] = EGL_CONTEXT_OPENGL_PROFILE_MASK_KHR;
       attribs[7] = EGL_CONTEXT_OPENGL_CORE_PROFILE_BIT_KHR;
       attribs[8] = EGL_NONE;
+#endif
     }
   else if (display->renderer->driver == COGL_DRIVER_GLES2)
     {
-- 
1.7.10.4

